ledger-service/http-json/src/main/scala/com/digitalasset/http/WebSocketService.scala

```diff
@@ -318,6 +318,7 @@ class WebSocketService(
       .mapAsync(1) {
         case msg: TextMessage =>
           msg.toStrict(config.maxDuration).map { m =>
+            logger.info(s"request: ${m.text}")
```

Comment:
Is this safe to log from a security/privacy standpoint?

Reply:
um... I don't actually know.
However if you get access to the disk that contains the log, means you have ssh access to the box, means you can access postgresql, and also tcp dump all traffic :)

Reply:
but going to ping Ed.

Reply:
It's fairly normal for ops teams to pipe logs to Logstash, Fluentd, StackDriver, etc, which means they might be more readily available, and not subject to any monitoring you might have about who's logged on to a box.

Reply:
true.

Reply:
@SamirTalwar thanks a lot for bringing this up... I moved the logging into tests for now, this would help to debug any further test failures/timeouts. As for the prod logging, this is going to be a separate discussion with @nycnewman